docs/rofl/prerequisites.md

```diff
@@ -17,6 +17,27 @@ If you already have everything set up, feel free to skip to the [next chapter].
 The following is a list of prerequisites required to start developing using the
 Oasis SDK:
 
+:::info
+
+Docker images are available to help you set up a development
+environment. If you don't want to install everything locally (or **in
+particular if you use macOS** as your development system), you can use
+the `ghcr.io/oasisprotocol/rofl-dev` image, which contains all the tools
+needed to compile a ROFL app.
+
+To use it, bind the directory with your app source to the container's
+`/src` directory with a command like the following, then continue with
+the next section of this guide:
+
+```bash
+docker run --platform linux/amd64 --volume ./rofl-oracle:/src -it ghcr.io/oasisprotocol/rofl-dev
```

Comment:
This command could actually be expanded in the application.md chapter to include full build commands.